src/DesktopVirtualization/Az.DesktopVirtualization.csproj

```diff
@@ -0,0 +1,44 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Do you force merge these files? On generation based branch, we don't submit csproj files. Because we'll auto generate it. Please double check **.gitignore** file and remove all unnecessary files.